File: samps/mini/config/sumit/submissions/security_violation/SumitSVFileWrite.pl
#
# File:    sumit.pl
# Purpose: to sum the integers - and write to a file  
# Author:  pc2@ecs.csus.edu or http://www.ecs.csus.edu/pc2
#
# Thu Nov  3 17:04:35 PDT 2016
#

	$sum = 0;

	while (<>)
	{
		chomp;
		$num = 0 + $_;
		$sum += $num if $num > 0;

	}

	$sum += 1000;

	print "The sum of the integers is $sum\n";

	open (FP, ">sumitplout.txt");
	print FP "The sum of the integers is $sum\n";
	close (FP);

# eof 

File: samps/contests/mini/config/sumit/submissions/security_violation/SumitSVFileWrite.c

/*
 * File:    isumit.c 
 * Purpose: to sum the integers from stdin
 * Author:  pc2@ecs.csus.edu or http://www.ecs.csus.edu/pc2
 *
 * Thu Nov  3 17:03:27 PDT 2016
 * 
 */

#include <stdio.h>

#define INFILENAME "sumit.dat"

int main (char **argv)
{
	int sum = 0;
	int i;

	while (1==fscanf(stdin, "%d", &i))
	{
		sum += i > 0 ? i : 0;
	}

	sum += 1000;

	printf("The sum of the integers is %d\n",sum);

	FILE *fpo = fopen ("sumitCoutputfile.txt", "w");
	fprintf(fpo, "The sum of the integers is %d\n",sum);
	fclose (fpo);

	return 0;
}

/* eof isumit.c */
